From f1048ea26c95fbb62358b7fbddb13f190d78bb0b Mon Sep 17 00:00:00 2001
From: Maxim Uvarov <maxim.uvarov@linaro.org>
Date: Thu, 25 Feb 2021 08:23:25 +0000
Subject: [PATCH] fix podman build

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>
---
 recipes-containers/podman/podman_git.bb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/recipes-containers/podman/podman_git.bb b/recipes-containers/podman/podman_git.bb
index aa6b25b..5e62880 100644
--- a/meta-virtualization/recipes-containers/podman/podman_git.bb
+++ b/meta-virtualization/recipes-containers/podman/podman_git.bb
@@ -12,6 +12,7 @@ DEPENDS = " \
     gpgme \
     libseccomp \
     ${@bb.utils.filter('DISTRO_FEATURES', 'systemd', d)} \
+    libselinux-native \
 "
 
 python __anonymous() {
-- 
2.17.1

